docs/en/help/contributing.md

```diff
@@ -135,7 +135,7 @@ When adding new functions or classes, include [Google-style docstrings](https://
         ❌ Bad - Don't document as a tuple with nested elements:
         ```
         Returns:
-           (tuple): Tuple containing:
+           (tuple): tuple containing:
```

Comment:
Ah, looks like we have a lot of accidental conversions here. First word of the sentence should stay capitalized. Maybe search and replace "): tuple" etc.